dbms/src/Disks/DiskLocal.cpp

```diff
@@ -101,7 +101,7 @@ bool DiskLocal::tryReserve(UInt64 bytes)
 
 UInt64 DiskLocal::getTotalSpace() const
 {
-    auto fs = getStatVFS(disk_path);
+    auto fs = getStatVFS(disk_path + "data/");
```

Comment:
`DiskLocal` used without `"data"` too, for `tmp_policy` (#8750 )
And there is no knowledge about `data` subfolder in the DiskLocal abstraction, maybe this knowledge should be added? (this way it can be used to auto append `tmp` for the temporary data too)

Reply:
Also does the `data` folder already when this method is called?

P.S. I just saw the WIP tag, sorry for the noise